src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/chips/ChipsRequest.java

```diff
@@ -89,7 +89,7 @@ public String toString() {
         for(String key : attachments.keySet()) {
             sb.append(String.format("%s:%s,", key, attachments.get(key)));
         }
-        String attachments = sb.toString().substring(0, sb.length()-1);
+        String attachments = sb.toString().substring(0, sb.length()-2);
```

Comment:
The need for this change goes away if we merge @markpit 's change - https://github.com/companieshouse/strike-off-objections-api/pull/75/files - into your branch.